decryption_utils/Python/decrypt.py

```diff
@@ -88,6 +89,8 @@ def valid_uuid(filename):
 
 def main():
     args = init()
+    if sys.version_info[0] < 3:
```

Comment:
Was this solution tested on Python 2.x _and_ Python 3.x ?

Reply:
yes vigorously.